ethers-core/src/types/signature.rs

```diff
@@ -324,6 +332,16 @@ mod tests {
         assert_eq!(tx.recover_from().unwrap(), expected);
     }
 
+    #[test]
+    fn can_recover_tx_sender_not_normalized() {
```

Comment:
This is from the Anvil test removed in https://github.com/foundry-rs/foundry/pull/6327 which I'll add back after this PR is merged

Reply:
https://github.com/foundry-rs/foundry/pull/6367